TGM/src/main/java/network/warzone/tgm/modules/LegacyDamageModule.java

```diff
@@ -16,19 +16,26 @@
     public void onDamage(EntityDamageByEntityEvent event) {
         if ((event.getDamager() instanceof Player)) {
             Player player = (Player) event.getDamager();
-            if (player.getInventory().getItemInMainHand().getType() == Material.DIAMOND_AXE) {
-                event.setDamage(event.getDamage() - 6.0D + 3.0D);
-            } else if (player.getInventory().getItemInMainHand().getType() == Material.IRON_AXE) {
-                event.setDamage(event.getDamage() - 6.0D + 2.5D);
-            } else if (player.getInventory().getItemInMainHand().getType() == Material.STONE_AXE) {
-                event.setDamage(event.getDamage() - 6.0D + 2.0D);
-            } else if ((player.getInventory().getItemInMainHand().getType() == Material.GOLD_AXE) ||
-                    (player.getInventory().getItemInMainHand().getType() == Material.WOOD_AXE)) {
-                event.setDamage(event.getDamage() - 4.0D + 1.5D);
-            } else if (player.getInventory().getItemInMainHand().getType() == Material.DIAMOND_SPADE) {
-                event.setDamage(event.getDamage() - 2.5D + 2.0D);
-            } else if (player.getInventory().getItemInMainHand().getType() == Material.STONE_SPADE) {
-                event.setDamage(event.getDamage() - 0.75D + 1.25D);
+            switch (player.getInventory().getItemInMainHand().getType()) {
```

Comment:
What happens if the sword is in their offhand?

Reply:
i only made the PR to make it work with 1.13, i didn't really change anything in terms functionality